packages/airview-cms/src/features/use-get-entry/use-get-entry.js

```diff
@@ -1,30 +1,18 @@
 import { useGetEntryQuery } from "../store";
-import { useGetAllEntriesMeta } from "../use-get-all-entries-meta";
+import { useGetCurrentBranch } from "../use-get-branches";
 
-export function useGetEntry(entryId) {
-  const { entryMetaData, isSuccess } = useGetAllEntriesMeta(
-    ({ data, isSuccess }) => ({
-      isSuccess,
-      entryMetaData: data?.find((entry) => entry.id === entryId),
-    })
-  );
-
-  const entryQuery = useGetEntryQuery(entryMetaData?.sha, {
-    skip: !entryMetaData?.sha,
-  });
+export function useGetEntry({ entryId, path }) {
```

Comment:
The `airview-cms` docs will need an update now the arguments have change for this hook